kernel-modules/container/Dockerfile

```diff
@@ -4,18 +4,20 @@ ARG base_image=${collector_repo}:${collector_version}-slim
 
 FROM python:3 as kernel-modules
 ARG max_layer_size=xxx
-ARG max_layer_depth=xxx
+ARG required_layers=xxx
 
 # Increase the value in this check if additional hardcoded layers are added below.
-RUN test "${max_layer_depth}" -le 9
+RUN test "${required_layers}" -le 9
```

Comment:
I don't really see the point in this check, I guess we were supposed to bring it back down when we dropped kernel-modules for it to make sense. I think we should either adjust the check so it warns when we go over the limit or just accept the number of layers can increase as a fact of life and get rid of the step altogether. Since we will be getting rid of more drivers in favor of CO-RE BPF, I'd get rid of the check.

Reply:
I left it in as a last check before we start copying from layers, but I suppose that's now controlled by the chosen target (and would probably give us a more useful error anyway.) I'll remove it.